jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannelState.java

```diff
@@ -740,13 +740,13 @@ protected void onError(Throwable th)
         final Request baseRequest = _channel.getRequest();
         
         int code=HttpStatus.INTERNAL_SERVER_ERROR_500;
-        String reason=null;
+        String message=null;
```

Comment:
not exactly, because we were treating some of the violation descriptions as reasons, which is not correct.  They should be treated as messages and this renaming makes that clear.  So a little out of scope... but near enough that I'm OK with including it.